components/post_view/message_attachments/message_attachment/message_attachment.jsx

```diff
@@ -242,9 +242,7 @@ export default class MessageAttachment extends React.PureComponent {
         );
     };
 
-    handleFormattedTextClick(e) {
-        handleFormattedTextClick(e, this.props.currentRelativeTeamUrl);
-    }
+    handleFormattedTextClick = (e) => handleFormattedTextClick(e, this.props.currentRelativeTeamUrl);
```

Comment:
can we change it like the others into Util.handleFormattedTextClick for the imported one so it is easy to differentiate between them?
I know it was like that earlier, but leaving it a bit more readable would be great.